cs-couchbase/src/main/java/io/cloudslang/content/couchbase/entities/couchbase/AuthType.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * (c) Copyright 2017 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.content.couchbase.entities.couchbase;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+/**
+ * Created by TusaM
+ * 4/14/2017.
+ */
+public enum AuthType {
+    NONE("none"),
+    SASL("sasl");
+
+    private final String value;
+
+    AuthType(String value) {
+        this.value = value;
+    }
+
+    public static String getAuthType(String input) throws RuntimeException {
```

Comment:
hmm - maybe, just maybe I would have a method `convertAuthType` that return AuthType - then you can invoke `getValue` on it (instead of having a mthod that directly return string)